src/components/TodoList.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from "react";
+import { useTodoStore, ITodo } from "../useTodoStore";
+import { arrayMove } from "@dnd-kit/sortable";
+import {
+  DndContext,
+  closestCenter,
+  DragEndEvent,
+  DragOverlay,
+  DragStartEvent,
+} from "@dnd-kit/core";
+import Droppable from "./Droppable";
+import TodoItem from "./TodoItem";
+import styled from "styled-components";
+
+const TodoLists = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+function TodoList() {
+  const todos = useTodoStore((state) => state.todos); // 기존 todo list 배열
+  const updateTodos = useTodoStore((state) => state.updateTodos); // todo list 순서 업데이트 함수
+  const updateCategory = useTodoStore((state) => state.updateCategory); // 카테고리 변경 시 todo list 순서 업데이트 함수
+  const [activeTodo, setActiveTodo] = useState<ITodo | null>(null);
+
+  const lists = [
+    {
+      id: "todo",
+      title: "To Do",
+      items: todos.filter((todo) => todo.category === "todo"),
+    },
+    {
+      id: "doing",
+      title: "Doing",
+      items: todos.filter((todo) => todo.category === "doing"),
+    },
+    {
+      id: "done",
+      title: "Done",
+      items: todos.filter((todo) => todo.category === "done"),
+    },
+  ];
+
+  const onDragStart = (event: DragStartEvent) => {
+    const { active } = event;
```

Comment:
```suggestion
  const onDragStart = ({ active }: Pick<DragStartEvent, 'active'>) => {
```
필요한 속성만 디스트럭처링으로 가져올 수 있고,

타입도 필요한 속성만으로 좁혀주는게
함수의 의도를 명확하게 전달하고, 함수의 책임을 제한할 수 있어 좋습니다